gitlab/metadata.csv

```diff
@@ -1,4 +1,162 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+gitlab.banzai.cached_render_real_duration_seconds.count,count,,second,,Count of duration of rendering Markdown into HTML when cached output exists,0,gitlab,cached render duration count
+gitlab.banzai.cached_render_real_duration_seconds.sum,gauge,,second,,Sum of duration of rendering Markdown into HTML when cached output exists,0,gitlab,cached render duration sum
+gitlab.banzai.cacheless_render_real_duration_seconds.count,count,,second,,Count of duration of rendering Markdown into HTML when cached output does not exist,0,gitlab,cachless render duration count
+gitlab.banzai.cacheless_render_real_duration_seconds.sum,gauge,,second,,Sum of duration of rendering Markdown into HTML when cached output does not exist,0,gitlab,cachless render duration sum
+gitlab.cache.misses_total,count,,second,,Cache read miss count,0,gitlab,cache read miss
```

Comment:
Since this metric is a prometheus counter and we're recording the metadata metric type as `count`, did you mean to update the `send_monotonic_counter` to True?

Reply:
The old version of the integration defaulted to `False`. Should I update it in the check?https://github.com/DataDog/integrations-core/pull/5971/files#diff-4c067887ce93cf3e795b69176c4cee4dR88

Reply:
Let's investigate if anyone is using the old legacy metrics to monitor gitlab first